client/a8c-for-agencies/components/form/fieldset/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { FormLabel } from '@automattic/components';
+import clsx from 'clsx';
+import { ReactNode } from 'react';
+import FormFieldset from 'calypso/components/forms/form-fieldset';
+import './style.scss';
+
+type Props = {
+	label: string;
+	error?: string;
+	required?: boolean;
+	description?: string;
+	children?: ReactNode;
+};
+
+export default function A4AFormFieldset( {
```

Comment:
This is created for the future, to have it aligned with all forms within A4A

Reply:
Why don't use it now?